Makefile

```diff
@@ -44,7 +44,7 @@ ARCH      := "`uname -s`"
 LINUX     := "Linux"
 MAC       := "Darwin"
 PACKAGE_LIST  := go list ./...| grep -vE "cmd"
-PACKAGES  := $$($(PACKAGE_LIST))
+PACKAGES  ?= $$($(PACKAGE_LIST))
```

Comment:
What's for this change?

Reply:
So that one can override PACKAGES and run tests for just a subset of packages, e.g., `PACKAGES=./store/tikv make gotest`